pkg/oc/cli/debug/debug.go

```diff
@@ -562,16 +569,24 @@ func (o *DebugOptions) getContainerImageCommand(pod *kapi.Pod, container *kapi.C
 		image, _ = o.getContainerImageViaImageStreamImport(container)
 	}
 
-	if image == nil || image.DockerImageMetadata.Config == nil {
+	// TODO: find a way to do this with the external api
+	// convert image to internal in order to extract its imagemetadata config
+	internalImage := &internalimage.Image{}
+	if err := oapiv1.Convert_v1_Image_To_image_Image(image, internalImage, nil); err != nil {
```

Comment:
@deads2k Is there a more elegant way to access an image's DockerImageMetadata.Config field? The external api stores the `DockerImageMetadata` field as a RawExtension

Reply:
We'll have to live with it for now.